components/wpt-results.html

```diff
@@ -146,40 +153,31 @@
         testFileResults.forEach(result => {
           const testFiles = result.testFiles
           const resultsURL = result.resultsURL
+          const encodedResultsURL = btoa(result.resultsURL)
```

Comment:
There was a reason I was encoding the results URL to base64 that I'm not remembering right now. But yes, that's correct.